playbooks/manage_clients_playbook.yml

```diff
@@ -0,0 +1,79 @@
+---
+
+- name: Gather facts from all hosts
+  hosts: all
+  become: true
+  tasks:
```

Comment:
Instead of this task, you can add the playbook parameter `gather_facts: true` and omit the `tasks` parameter.

Reply:
thx! is added.